easybuild/easyconfigs/f/FFTW/FFTW-3.3.6-intel-2017a.eb

```diff
@@ -0,0 +1,29 @@
+name = 'FFTW'
+version = '3.3.6'
+
+homepage = 'http://www.fftw.org'
+description = """FFTW is a C subroutine library for computing the discrete Fourier transform (DFT)
+ in one or more dimensions, of arbitrary input size, and of both real and complex data."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+source_urls = [homepage]
+sources = ['fftw-%(version)s-pl1.tar.gz']
+checksums = ['682a0e78d6966ca37c7446d4ab4cc2a1']
+
+patches = [
+    'FFTW-3.3.6-F03_interface_pl2.patch',
+]
+
+# no quad precision, requires GCC v4.6 or higher
+# see also
+# http://www.fftw.org/doc/Extended-and-quadruple-precision-in-Fortran.html
+with_quad_prec = False
+
+# compilation fails when configuring with --enable-avx-128-fma
+use_fma = False
```

Comment:
It failed when i tried to use it, i.e., when i tried use_fma=true.
Haven't seen an issue about it in fftw issues though.